client/my-sites/checklist/wpcom-checklist/checklist-banner/style.scss

```diff
@@ -1,8 +1,7 @@
-
 .checklist-banner {
 	display: flex;
 	flex-direction: row;
-	background: #daf5fb;
+	background: $muriel-blue-0;
```

Comment:
Please use `var( --color-primary-0 )` at this place. It will make sure we can adjust those color from a theming perspective. You can find all available colors in `_color-schemes.scss` (available on the [`feature/color-refresh-2019` branch here](https://github.com/Automattic/wp-calypso/blob/feature/color-refresh-2019/assets/stylesheets/shared/_color-schemes.scss#L9)).

Background context is that we want to aggregate any color scheming related changes in this feature branch and merge them altogether.